src/elements/common/feature-checking/__tests__/FeatureChecking.test.js

```diff
@@ -76,6 +76,46 @@ describe('FeatureFlag', () => {
         expect(enabledFn).toHaveBeenCalledWith(foo);
         expect(disabledFn).toHaveBeenCalled();
     });
+    test('"not" prop inverts flag with children prop', () => {
```

Comment:
Can we add a test for the explicit `features={{ foo: false }}` and `not feature="foo"` case?